src/main/js/controls/PairedResult/PairedResult.js

```diff
@@ -191,17 +192,43 @@ class PairedResult extends GraphContent {
      */
     resize(graph) {
         if (utils.notEmpty(this.dataTarget.regions)) {
-            // If graph has more than 1 content then we dont load the regions
+            // If graph has more than 1 content, we compare the regions if they are identical and hide if even atleast one of them is not.
+            const regionList = this.dataTarget.regions;
+            const values = this.dataTarget.values;
+            //check if all region are there with respect to value (high, mid and low)
             if (graph.content.length > 1) {
-                hideAllRegions(graph.svg);
+                const isRegionThere = (value) => {
+                    for (const key in value) {
+                        if (!regionList.hasOwnProperty(key)) {
+                            return false;
+                        }
+                    }
+                    return true;
+                };
+                const isPairedDataProper = values.every(isRegionThere);
```

Comment:
I have already added one test case for this for multi-paired with missing region.

Reply:
Which unit test is that?

Reply:
->  it("Hide region if region is missing for any value(high, mid, low)") 

line no - 3204

Reply:
Doesnt all unit test pass even with the condition and every line associated to `isPairedDataProper` removed? 

```js
if (graph.content.length > 1) {
    if (
        graph.config.shouldHideAllRegion === false &&
        areRegionSame(graph.svg)
    ) {
        graph.config.shouldHideAllRegion = false;
    } else {
        hideAllRegions(graph.svg);
        graph.config.shouldHideAllRegion = true;
    }
}
```

![image](https://user-images.githubusercontent.com/4908886/62464847-41770b00-b753-11e9-9f6e-04d6588ea1f7.png)

Reply:
it is because the condition i have given for regionMissing in that region are not same so areRegionsSame is returning false so that is why the test cases are  passing if you change the parameters  of regionMissing to same as the multiRegionSameData then this will fail in that case.

Reply:
So you are going to fix the test? Sorry, what is the resolution here?

Reply:
it is working properly, no fix is need for this.

Reply:
I made some changes to the test case